gen/nvml/const_gen.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Nit: Should we update the year in the license string?

Reply:
Updated.